### PR TITLE
AENV-487: Save button is not active after unro/redo action with image

### PR DESCRIPTION
**Pull request by thnkloud9**
fixed

### CODE REVIEW

File: app/scripts/controllers/articleActions.js [hunk 1]
```diff
@@ -52,7 +52,13 @@ angular.module('authoringEnvironmentApp').controller('ArticleActionsCtrl', [
...
             eventObj, jqEvent, alohaEditable
         ) {
-            var reactOnTypes = {'keypress': true, 'paste': true, 'idle': true};
+            var reactOnTypes = {
+                'keypress': true,
+                'paste': true,
+                'idle': true,
+                'undo': true,
+                'redo': true
+            };
```
Comment (takeit) [inline app/scripts/controllers/articleActions.js:61]: what about tests here ?

File: app/scripts/controllers/article.js [hunk 2]
```diff
@@ -66,7 +66,13 @@ angular.module('authoringEnvironmentApp').controller('ArticleCtrl', [
...
             var fieldName,
                 statsText,
-                reactOnTypes = {'keypress': true, 'paste': true, 'idle': true};
+                reactOnTypes = {
+                    'keypress': true,
+                    'paste': true,
+                    'idle': true,
+                    'undo': true,
+                    'redo': true
+                };
```
Comment (takeit) [inline app/scripts/controllers/article.js:75]: what about tests here ?

File: app/scripts/controllers/articleActions.js (hunk 1 above)
Comment (thnkloud9) [inline app/scripts/controllers/articleActions.js:61]: yeah, thought it might be overkill, but you are right.

**Comment by takeit** [comment]
:+1:

**Pull request merged**